### PR TITLE
[WIP] Added ZoeDepth Support to SHARK

### PR DESCRIPTION
**Pull request by AyaanShah2204**
```
python apps/stable_diffusion/scripts/img2img.py  --precision=fp16 --device=vulkan --prompt="barbie pink dog" --max_length=64 --import_mlir iree-hal-dump-executable-sources --enable_stack_trace --no-use_tuned  --img_path=C:\Users\nithin\intern_stuff\SHARK\wolf.jpg --use_stencil="zoedepth"
```
This will use the stable diffusion stencil pipeline to create a depth map of the input image and then stay faithful to the depth map when generating new images.

### CODE REVIEW

File: apps/stable_diffusion/src/pipelines/pipeline_shark_stable_diffusion_stencil.py [hunk 1]
```diff
@@ -135,6 +135,7 @@ def produce_stencil_latents(
                 ).to(dtype)
             else:
                 latent_model_input_1 = latent_model_input
+
```
Comment (powderluv) [inline apps/stable_diffusion/src/pipelines/pipeline_shark_stable_diffusion_stencil.py:138]: white space

**Review by powderluv — CHANGES_REQUESTED** [review]

File: apps/stable_diffusion/src/pipelines/pipeline_shark_stable_diffusion_stencil.py (hunk 1 above)
Comment (AyaanShah2204) [reply apps/stable_diffusion/src/pipelines/pipeline_shark_stable_diffusion_stencil.py]: done

**Comment by AyaanShah2204** [comment]
Changing to WIP. Looks like gitignore prevents `models` directory within zoe from being added. checking if I can move all the references to the `models` module into a utils file

**Comment by monorimet** [comment]
Commit 72e27c96fce932b5b31f708af19a67d59a53d438 adds this support. Closing this PR.

**Pull request closed without merging**